lib.php

```diff
@@ -4420,6 +4420,22 @@ function facetoface_can_view_field(int $visibility, bool $editsessions): bool {
     }
 }
 
+function facetoface_enrol_user($context, $courseid, $userid) {
```

Comment:
@djarran could you please add phpdoc to this function? Please add it as a new commit to facilitate the following review. Thanks